server/athenian/api/openapi/openapi.yaml

```diff
@@ -1357,7 +1434,8 @@ components:
           description: Number of commits in this PR.
           type: integer
         review_requested:
-          description: Value indicating whether the author of this PR requested a review.
+          description: Value indicating whether the author of this PR requested a
```

Comment:
Only the author?
What if any other contributor requested a review? is it not considered for the `review_requested`?
Imo it should not depend on the authorship of the request.
(it could be made automatically by a bot, assigning a reviewer if the PR meets some requirements)

Reply:
OK, I forgot about it :+1: